feign-reactor-core/src/test/java/reactivefeign/BaseReactorTest.java

```diff
@@ -19,14 +19,15 @@
 
     @BeforeClass
     public static void installBlockHound() {
-        if(INSTALL_BLOCKHOUND) {
+        if (INSTALL_BLOCKHOUND) {
             BlockHound.Builder builder = BlockHound.builder();
             ServiceLoader<BlockHoundIntegration> serviceLoader = ServiceLoader.load(BlockHoundIntegration.class);
             serviceLoader.forEach(builder::with);
 
             //spring
             //check fails on server side as MimeTypeUtils$ConcurrentLruCache use this.lock.readLock().lock();
             builder.allowBlockingCallsInside("org.springframework.util.MimeTypeUtils", "parseMimeType");
+            builder.allowBlockingCallsInside("org.springframework.web.reactive.function.BodyExtractors", "readWithMessageReaders");
```

Comment:
these exceptions looks odd to me
```org.springframework.web.reactive.function.BodyExtractors``` blocking?

Reply:
I came across that when it was calling down in Jackson. No idea why.

But I'm also struggling to execute the test cases locally, so reverted, and hope that CI can see whats happening